pyomo/addons/README.md

```diff
@@ -0,0 +1,58 @@
+# Pyomo Addons
+
+The `pyomo.addons` directory contains **mostly stable extensions** to Pyomo
+that are **not part of the core library*. These packages extend Pyomo's
+modeling, solver, and analysis capabilities, and have demonstrated sufficient
+maturity, documentation, and testing.
+
+## Purpose
+
+`pyomo.addons` houses packages that:
+
+- Have **mostly stable APIs** suitable for downstream use.
+- Are **sufficiently documented** and **tested**.
+- Represent functionality that is **useful to the wider Pyomo community** but
+  not required for core operation.
+
+Examples include:
+- `gdpopt`
+- `incidence_analysis`
+- `latex_printer`
+- `trustregion`
+- `viewer`
```

Comment:
can't people just look to see what is in the addons directory? Why list it here?